website/docs/reference/projects.md

```diff
@@ -3,131 +3,119 @@ id: projects
-| Project Id         | Id for this Project                                                                         |
-| Project name       | The name of the Project.                                                                    |
-| Description        | A short description of the project                                                          |
-| Mode               | The project [collaboration mode](/reference/project-collaboration-mode.md)                  |
-| Default Stickiness | The default stickiness for the project. This setting controls the default stickiness value for variants and for the gradual rollout strategy.                                                                 |
+You can update all aspects of a project except its ID.
 
-## Deleting an existing project {#deleting-an-existing-project}
+To update a project, go to **Projects** and select the project you'd like to edit. In the **Project settings** tab, you can update general settings such as project name, description, [stickiness](/reference/stickiness.md), [collaboration mode](/reference/project-collaboration-mode.md), and more. You can also configure user and API access, [change requests](/reference/change-requests.md), and [actions](/reference/actions.md), add [segments](/reference/segments.mdx) and [environments](/reference/environments.md), and update the [default strategy](#project-default-strategy).
 
-To keep your feature flags clean, removing deprecated projects is important. From the overview of Projects –
+The available project settings depend on a user's [root and project roles](/reference/rbac).
 
-1. In the top right of the project card, find the project menu represented by three vertical dots.
+## Archive a project
 
-![A list of projects. Each project has three vertical dots — a kebab menu — next to it.](/img/projects_menu_button.png)
+:::note Availability
 
-2. Click on Delete Project
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing). | **Version**: 6.?+ in BETA.
 
-![A list of projects. Each project has three vertical dots — a kebab menu — next to it, which exposes a menu with the "Edit project" and "Delete project" options when interacted with.](/img/projects_delete_button.png)
+:::
 
-## Filter feature flags on projects {#filter-feature-toggles-on-projects}
+The archive projects functionality allows you to hide a project while maintaining historical project data. Archiving allows you to have short-lived projects, for example, to represent specific initiatives in your organization.
 
-When browsing the feature flags in Unleash, you might want to filter the view by looking only at the ones included in the project of interest. This is possible from the Feature flag overview.
+Before archiving a project, you must archive all feature flags within it. Once you archive a project, you cannot add or move feature flags to it.
```

Comment:
We also automatically disable all project actions (https://docs.getunleash.io/reference/actions) when the project is archived. Maybe it's worth adding here or in the actions docs.